tests/test_cli.sh

```diff
@@ -10,8 +10,9 @@ set -eu
 CLI="${CLI:-}"
 
 function setup_tests {
+    [ "$BACKEND" == "osbuild-composer" ] && return 0
```

Comment:
Related to all the above changes from example-http-server.toml to test-http-server.toml wouldn't it be cleaner to instead have osbuild-composer push the copy of the expected example blueprints (example-http-server, example-development, example-atlas) from tests/pylorax/blueprints/? That way there's less change to account for.

Reply:
In the general case of working with osbuild-composer we don't have access to the pylorax source code so I'd say no. It can be done ATM but will require changes very soon IMO.